### PR TITLE
Add a test that `zypper up` works in every container image

### PR DESCRIPTION
**Pull request by dcermak**

### CODE REVIEW

File: tests/test_all.py [hunk 1]
```diff
@@ -101,6 +103,14 @@ def test_glibc_present(auto_container):
         assert auto_container.connection.exists(binary)
 
 
+@pytest.mark.parametrize(
+    "container_per_test", CONTAINERS_WITH_ZYPPER, indirect=True
+)
+def test_zypper_up_works(container_per_test: ContainerData) -> None:
+    """Check that :command:`zypper -n up` works."""
+    container_per_test.connection.run_expect([0], "zypper -n up")
```
Comment (jlausuch) [inline tests/test_all.py:111]: btw, does this command have a default timeout?

**Review by jlausuch — DISMISSED** [review]

**Comment by dirkmueller** [comment]
Why are we doing this? What about container without zypper? I don't think this makes sense for application containers

File: tests/test_all.py (hunk 1 above)
Comment (dcermak) [reply tests/test_all.py:111]: I don't think it has one

**Comment by dcermak** [comment]
> Why are we doing this?

This idea came up during our containers doc sync. A `zypper -n ref && zypper -n up` is a very common pattern in dockerfiles, so we should be testing that it actually works and isn't broken in some subtle way.

> What about container without zypper?

This test only runs on all containers in the list `CONTAINERS_WITH_ZYPPER`.

>  I don't think this makes sense for application containers

I don't have an opinion on this tbh.

File: tests/test_all.py (hunk 1 above)
Comment (dirkmueller) [reply tests/test_all.py]: I added a `timeout`

**Review by dirkmueller — DISMISSED** [review]

**Pull request merged**